src/pages/CreateBurnerWallet.tsx

```diff
@@ -52,6 +53,13 @@ const buttonWithStatus = classnames(
 )
 const buttonClass = classnames(display('block'), width('w-full', 'sm:w-64'))
 
+const getAxiosErrorMessage = (error: unknown) => {
```

Comment:
Should we extract it to a separate file?

Reply:
Moved it to the separate file